src/odemis/dataio/tiff.py

```diff
@@ -2722,3 +2752,210 @@ def _convertTFStoOdemisMD(metadata: str) -> dict:
         logging.warning(f"Failed to parse ThermoFisher metadata: {model.MD_DET_TYPE} - {e}")
 
     return md
+
+def _parseZEISSMetadata(metadata) -> dict:
+    """
+    Parses ZEISS metadata to dictionary
+    :param metadata: metadata string
+    :return: (dict) metadata as dictionary
+    """
+    zmd = {}
+    for line in metadata.split("\r\n"):
+
+        # Metadata is in the format:
+        # HEADER
+        # key = value unit
+
+        # Example:
+        # AP_STAGE_AT_X
+        # Stage at X = 48.5090 mm
+
+        # TODO: try to do it similar to tifffile to extract header information too            
+        try:
+            # NOTE: 
+            # Date, Time are separated by : 
+            # everything else is separated by =
+            try:
+                k, v = line.split("=")
+            except:           
+                k, v = line.split(":", maxsplit=1) 
+            k, v = k.strip(), v.strip()
+            zmd[k] = v
+        except Exception as e:
+            continue
+
+    return zmd
+
+
+def _convertZEISStoOdemisMD(metadata: str) -> dict:
+    """Converts ZEISS metadata to odemis metadata format
+    :param metadata: metadata string
+    :return: (dict) metadata as dictionary
+    :raises: (KeyError): if the metadata cannot be parsed
+    """
+
+    # parse as dict
+    zmd = _parseZEISSMetadata(metadata)
+
+    md = {}
+
+    # convert to odemis format
```

Comment:
I think you can drop this comment. This is essentially what the whole function does!